### ISSUE TITLE
Package as flatpak and make it available on Flathub

### ISSUE THREAD

**Issue by hyuri** [ggarra13/mrv2#54]
Hi!

It would be great to have mrv2 available as a flatpak on [Flathub](flathub.org), to make it easier for people to discover and install it.

Flathub is probably the biggest Linux app hub right now, with over 2,000 apps available and over 700,000 downloads every day. And newly-added apps show up at the top of the home page, in the "New Apps" section.

Additionally, Linux-related blogs and vlogs often advertise apps newly-added to Flathub, which could promote mrv2 even further.

**Comment by ggarra13** [comment]
My understanding is that flatpaks run in a sand-box, so they cannot access things like the installed NVidia or AMD drivers.  They would have to be distributed with them and, my belief is, that that would break their EULA or, at the very least, it would make it difficult to choose between one driver or the other.

**Comment by hyuri** [comment]
They are available under the freedesktop runtime, if I understand it well.

For example:
https://github.com/flathub/org.freedesktop.Platform.GL.nvidia

For reference:
https://docs.flatpak.org/en/latest/available-runtimes.html

**Comment by ggarra13** [comment]
Thanks for the links.  I found this:

https://blogs.igalia.com/vjaquez/2022/01/17/digging-further-into-flatpak-with-nvidia/

which leads me to believe supporting the nvidia driver is non-trivial.  For an application like mrv2 which depends on many libraries, like pulse audio, python, and others that are compiled when building the application with cmake, it is not an easy change.  I tried looking at VLC's flatpak manifest as a reference and it had to list each and every library it uses to compile (somthing I would like to avoid).

I read Ubuntu's snaps are supossed to vanish in the next release so it will be interesting to see if Cannonical chooses Flatpak (A RH invention) or AppImage (an independent creation) as its sucessor, so it may make sense to wait until then. 

I did not try Flatpak yet, but I did try AppImage.  Both Flatpak and AppImage depend on the fuse library, which created havok when I installed it on my dual-boot machine, removing all /etc/fstab ntfs-3g mounts.
